faf.spec.in

```diff
@@ -816,9 +816,6 @@ fi
 %config(noreplace) %{_sysconfdir}/faf/plugins/fedora.conf
 %{python_sitelib}/pyfaf/opsys/fedora.py*
 
-%files opsys-rhel
-%{python_sitelib}/pyfaf/opsys/rhel.py*
```

Comment:
As this is only one file in subpackage opsys-rhel, this can be safely removed (see top of spec file)